### PR TITLE
Add coursera-dl recipe

### PR DESCRIPTION
**Pull request by proinsias**
- [x] Tidy up `meta.yaml` after build succeeds

### CODE REVIEW

File: recipes/coursera-dl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+test:
+  imports:
+    - coursera
+
+about:
+  home: https://github.com/coursera-dl/coursera-dl
+  license: LGPLv3+
+  license_file: LICENSE
+  license_family: Public-Domain
```
Comment (jakirkham) [inline recipes/coursera-dl/meta.yaml]: That can't be right. 😜 Maybe LGPL? 😉

**Comment by proinsias** [comment]
@jakirkham – updated to LGPL-3.0

File: recipes/coursera-dl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+test:
+  imports:
+    - coursera
+
+about:
+  home: https://github.com/coursera-dl/coursera-dl
+  license: LGPL-3.0
+  license_file: LICENSE
+  license_family: Public-Domain
```
Comment (jakirkham) [inline recipes/coursera-dl/meta.yaml]: `LGPL`?

File: recipes/coursera-dl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - beautifulsoup4 >=4.1.3
+    - requests >=2.10.0
+    - six >=1.5.0
+    - urllib3 >=1.10
+    - pyasn1 >=0.1.7
+    - keyring >=4.0
+
+test:
+  imports:
+    - coursera
```
Comment (jakirkham) [inline recipes/coursera-dl/meta.yaml]: Could we please test the entrypoint like so? A different option can be used if needed.

```yaml
  commands:
    coursera-dl --help
```

**Comment by proinsias** [comment]
@jakirkham – I've addressed both requests. Sorry about the LGPL part – totally missed it.

**Comment by proinsias** [comment]
setuptools is needed by the build, apparently.

File: recipes/coursera-dl/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipes/coursera-dl/meta.yaml]: Note: Done below.

**Comment by jakirkham** [comment]
LGTM. Any reason it is still marked WIP?

**Comment by proinsias** [comment]
@jakirkham – just wanted to make sure `meta.yaml` was tidy. All ready for a merge.

**Comment by jakirkham** [comment]
Makes sense. Thanks @proinsias .

**Pull request merged**